src/git-sync

```diff
@@ -31,6 +31,13 @@ function preconditions {
 }
 
 
+function set_previous_branch {
+  if [ "$IN_SUB_FOLDER" = false ]; then
+    set_initial_previous_branch_as_previous_branch
```

Comment:
Yes you're right (see line 88). This was left behind from a previous commit. Will remove the function.